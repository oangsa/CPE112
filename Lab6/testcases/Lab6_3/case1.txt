7
4
L 2
LL 1
LR 3
R 6
RL 5
RR 7
