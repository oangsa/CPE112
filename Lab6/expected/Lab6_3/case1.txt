4 3
4 3
3 2
5 1
Queue is empty.
Queue is empty.
